UN-OCHA/rwint9-site
# [OPS-10081] OCHA AI module

Refs: OPS-10081

This adds the new OCHA AI module and enables the AI "chat". (The job tagging submodule is not enabled in that PR.)

## Tests

1. Check https://humanitarian.atlassian.net/browse/RW-857?focusedCommentId=166569 for local install.
2. Import a recent DB light dump
3. Clear the cache, import the config
4. Once your local RW API and site are setup, visit a report page, there should be a chat button on the bottom right, click on it and ask questions.
